apps/src/lab2/types.ts

```diff
@@ -209,6 +209,12 @@ export interface LevelProperties {
   miniApp?: string;
   serializedMaze?: MazeCell[][];
   startDirection?: number;
+  // Properties added for parity with non-lab2 AI Tutor levels
+  aiTutorAvailable?: boolean;
+  hasValidation?: boolean;
+  isAssessment?: boolean;
+  progressionType?: string;
+  type?: string;
```

Comment:
`type` wasn't on LevelProperties yet, but was already in `summarize_for_lab2`: https://github.com/code-dot-org/code-dot-org/blob/b49cc801f82b45fa443ece4c5ca91cb5d1125c19/dashboard/app/models/levels/level.rb#L852-L853